src/emulator/pubsubEmulator.ts

```diff
@@ -62,7 +70,21 @@ export class PubsubEmulator implements EmulatorInstance {
   }
 
   async stop(): Promise<void> {
-    await downloadableEmulators.stop(Emulators.PUBSUB);
+    try {
+      await downloadableEmulators.stop(Emulators.PUBSUB);
+    } catch (e: unknown) {
+      this.logger.logLabeled("DEBUG", "pubsub", JSON.stringify(e));
+      if (process.platform === "win32") {
```

Comment:
This check looks like the opposite of what you wanted. Do you want `!==`?

Reply:
Correct thanks for pointing it out